mods/Example.Mod/mod.json

```diff
@@ -4,6 +4,7 @@
 	"LoadPriority": 2,
 	"RequiredOnClient": true,
 	"Version": "0.0.1",
+	"DownloadLink": "https://northstar.thunderstore.io/",

```

Comment:
Is `DownloadLink` just the hoster domain or link directly to package, e.g. https://northstar.thunderstore.io/package/Fifty/NorthstarLoadingAnim/